### ISSUE TITLE
[PS2] Is there a way to render in 240p?

### ISSUE THREAD

**Issue by 60fpshacksrock** [fgsfdsfgs/sm64-port#73]
Just what it says above. 

It would look really nice on CRT's.

**Comment by fgsfdsfgs** [comment]
On the PS2 version?

**Comment by 60fpshacksrock** [comment]
Yes. 

My apologies for not stating that in the title.

**Comment by fgsfdsfgs** [comment]
I don't think anything lower than 640x480 is supported by gsKit, so there's no easy way to do that.

**Comment by 60fpshacksrock** [comment]
Thanks again.

Edit: Check this out.
https://github.com/ps2dev/gsKit/pull/30

Also, I think retroarch supports 240p on PS2 does it not?

**Comment by fgsfdsfgs** [comment]
Probably has that PR merged then. Or doesn't use gsKit.

**Comment by wolfmanvid01** [comment]
yes... 240p solved [here](https://github.com/fgsfdsfgs/sm64-port/issues/94)
